otj-metrics-actuator/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.opentable.metrics.actuate.micrometer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(OtMicrometerConfig.CONFIGURATION_PREFIX)
```

Comment:
Can you please add more comments. I guess this is basically turning the duplication off by default

Reply:
yeah, just to reuse config in both contributors.